lib/setupmanagers/qemuhck/states.json

```diff
@@ -49,7 +54,9 @@
     },
     "drive_unsafe_cache_state": {
         "true": {
-            "drive_cache_options": ",cache=unsafe"
+            "drive_cache_options": [
+                ",cache=unsafe"
```

Comment:
```suggestion
                "cache=unsafe"
```

Reply:
Unfortunately, no. We use `@drive_cache_options@` in `"-drive file=@image_path@,if=none,id=ide_@run_id@_@client_id@@drive_cache_options@"` and should somehow add `,` before drive_cache_options if it present.

Reply:
In any case we need some if/map/etc to add ',' before all options but only if options is not empty